src/components/YextEntityFieldSelector.tsx

```diff
@@ -127,18 +127,18 @@ const ToggleMode = ({
         <div className="ve-flex ve-items-center ve-space-x-2">
           <RadioGroupItem
             value="false"
-            id="r1"
+            id="false"
```

Comment:
It needs an actual id. Ids are unique for DOM elements. I'd call it something like ve-use-entity-value and ve-use-constant-value

Reply:
Ohh forgot about that. Fixed!